tests/test_tacacs_server_host.rb

```diff
@@ -147,16 +166,19 @@ def test_tacacsserverhost_get_port
     port = DEFAULT_TACACS_SERVER_HOST_PORT
     assert_equal(port, host.port, 'Error: Tacacs Host port incorrect')
 
-    # when configured
-    port = 1138
-    config("tacacs-server host #{@host_name} port #{port}")
-    assert_equal(port, host.port, 'Error: Tacacs Host port incorrect')
+    # IOS XR support the same host using different ports
```

Comment:
I can't think of anything to change this to. IOS support having the same host using different ports, so this means that instead of updating the port, it just adds another host.